packages/components/credentials/SearchApi.credential.ts

```diff
@@ -0,0 +1,24 @@
+import { INodeParams, INodeCredential } from '../src/Interface'
+
+class SearchApi implements INodeCredential {
+    label: string
+    name: string
+    version: number
+    description: string
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'Search API'
+        this.name = 'searchApi'
+        this.version = 1.0
+        this.inputs = [
```

Comment:
would be great if we can add a `description` on how to get the API key. Example [here](https://github.com/FlowiseAI/Flowise/blob/main/packages/components/credentials/AWSCredential.credential.ts#L15)

Reply:
Thanks, that's a great suggestion!